### ISSUE TITLE
[Permssion] Add "View own site DICOM archive" permission

### ISSUE THREAD

**Issue by cmadjar** [aces/Loris#5033]
In the user account module, there is no "View own site DICOM archive" permission. Only a "View all site DICOM archive" permission.

Need to add the "View own site DICOM archive" permission in User account module so that users can see it and update the code for the DICOM archive module to use that new permission.

**Comment by christinerogers** [comment]
This was originally reported in RM # 8461

**Comment by johnsaigle** [comment]
Could this be addressed by using the Site filter in the module?

**Comment by cmadjar** [comment]
@johnsaigle yes, I guess we could use the site filter but a "View own site DICOM archive" permission still need to be added to the permissions unless I am missing something obvious... :S

**Comment by cmadjar** [comment]
permission still does not exist.
